screens/user-list.tsx

```diff
@@ -30,12 +30,26 @@ export const UserListScreen = () => {
     onCompleted: onCompleted,
   });
 
+  const handleButtonPress = (id: string) => {
+    Navigation.push(props.componentId, {
+      component: {
+        name: 'UserDetails',
+        passProps: {
+          id: id,
+        },
+      },
+    });
+  };
+
   const renderItem = (item: { item: UserType }) => {
     return (
-      <>
+      <View style={styles.user}>
         <Text style={styles.item}>{`Nome : ${item.item.name}`}</Text>
         <Text style={styles.item}>{`Email : ${item.item.email}`}</Text>
-      </>
+        <Button mode='outlined' onPress={() => handleButtonPress(item.item.id)}>
+          {'Ver perfil'}
```

Comment:
```suggestion
          Ver perfil
```

Quando é a string direta como child n precisa do `{''}`

Reply:
Feito!